tests/basic/misc_test.go

```diff
@@ -1784,4 +1784,71 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
```

Comment:
I feel that creating a function which returns the PID of Px for the given nodes will help reduce the code.

Reply:
`log.Infof` accepts format arguments just like `fmt.Printf`, so there is no need to form a string using `fmt.Sprintf` before passing it to `log.Infof`.

Reply:
How about we use `PodDeletion` or anything similar instead of `PodStop`?

Reply:
I do not think to have separate function for PID, anyways it is a single line code though it looks multiline

Reply:
I was thinking of a function like below

```go
// GetPxPIDMap returns a map of node ID to PX PID
func GetPxPIDMap(nodes []node.Node) (map[string]string, error) {
	pxPIDMap := make(map[string]string)
	getPxPidCmd := "pidof px"

	for _, n := range nodes {
		err := Inst().V.WaitForPxPodsToBeUp(n)
		if err != nil {
			return nil, fmt.Errorf("failed to wait for PX pod to be up in node [%s]. Err: [%v]", n.Name, err)
		}
		output, err := Inst().N.RunCommand(n, getPxPidCmd, node.ConnectionOpts{Timeout: 30 * time.Second, TimeBeforeRetry: 20 * time.Second, Sudo: true})
		if err != nil {
			return nil, fmt.Errorf("failed to get PX PID on node [%s]. Err: [%v]", n.Name, err)
		}
		pxPIDMap[n.Id] = output
	}
	return pxPIDMap, nil
}
```